blocks/text.js

```diff
@@ -825,18 +830,24 @@ Blockly.Constants.Text.TEXT_CHARAT_MUTATOR_MIXIN = {
         this.appendDummyInput('ORDINAL')
```

Comment:
!== '' to avoid problems with falsy strings.  (line above this one)

Reply:
'' is the only falsy string.  The existing code is correct (and it also has the advantage of catching undefined).